src/Css/Processor.php

```diff
@@ -24,6 +24,21 @@ public function getRules($css, $existingRules = array())
         return $rulesProcessor->convertArrayToObjects($rules, $existingRules);
     }
 
+    /**
+     * Get all media queries from the given css string. Returns an empty array
+     * when non were found.
+     *
+     * @param string $css
+     * @return array
+     */
+    public function getMediaQueries(string $css)
+    {
+        if (preg_match_all('/@media [^{]*+{([^{}]++|{[^{}]*+})*+}/', $css, $matches)) {
+            return $matches[0];
+        }
+        return [];
```

Comment:
this library is not using the short array syntax currently. Please use `array()` for consistency.